packages/api/src/server/analytics/service/analytics.factory.ts

```diff
@@ -78,4 +109,11 @@ export class AnalyticsFactory {
       { $sort: { env: 1 } }
     ];
   }
+
+  async getDateDiffrence(days: number): Promise<Date[]> {
```

Comment:
as per my understanding this would be nicer
`async startAndEnd(difference: number): Promise<Date[]>{
      ...
}`

@soumyadip007

Reply:
sure @arkaprovob  I'll make it getStartAndEndDate()